src/components/ChatEntry.js

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ sender, body, timeStamp, liked, onLikeClick }) => {
+  const isLocal = sender === 'Vladimir';
```

Comment:
A good thought exercise here might be thinking about what you'd have to change in order to avoid hard-coding sender to a particular name.